dandischema/models.py

```diff
@@ -820,26 +880,36 @@ class CommonModel(DandiBaseModel):
         nskey="schema",
     )
     studyTarget: Optional[List[str]] = Field(
-        None, description="What the study is related to", nskey="dandi"
+        None,
+        description="What the objectives or specific questions of the study are.",
+        nskey="dandi",
     )
     license: Optional[List[LicenseType]] = Field(
-        None, description="Licenses associated with the item.", nskey="schema"
+        None,
+        description="Licenses associated with the item. DANDI only supports a "
+        "subset of Creative Commons Licenses (creativecommons.org) "
+        "relevant to datasets.",
+        nskey="schema",
     )
     protocol: Optional[List[HttpUrl]] = Field(
-        None, description="A list of protocol.io URLs", nskey="dandi"
+        None,
+        description="A list of persistent URLs describing the protocol (e.g. "
+        "protocols.io, or other dois).",
```

Comment:
```suggestion
        "protocols.io, or other DOIs).",
```
for consistent use of DOI abbreviation (and above `URLs`)